<launch>
    <param name="broker_address"    value="10.14.7.175" />
    <param name="broker_port"       value="1883" />
    <param name="username"          value="mascot" />
    <param name="password"          value="demo" />

    <param name="mqtt_sub_points"        value="robot/send_points" />
    <param name="mqtt_pub_waypoints"        value="robot/waypoints" />
    <param name="mqtt_pub_pose"        value="robot/pose" />
    <param name="mqtt_pub_estimate"        value="robot/estimate" />    
    <param name="mqtt_sub_move"        value="robot/move" />
    <param name="cwt_pub_mqtts"        value="env/cwt_sensor" />    
   

    <node name="waypoints" pkg="ros_mqtt" type="waypoints.py" output="screen"/>    
    <node name="calc" pkg="ros_mqtt" type="cal_distance.py" output="screen"/>   
    <node name="goal_plan" pkg="ros_mqtt" type="send_goal.py" output="screen"/>
    <node name="swt_sensor" pkg="ros_mqtt" type="cwt_sensor.py" output="screen"/>

</launch>
